dimagi/pages/static/style/pages/open_source/open_source_home.scss

```diff
@@ -0,0 +1,90 @@
+@import 'app/mixins/background';
```

Comment:
This would be a good file to review in full. I wasn't sure how much custom styling was ok to apply to particular elements versus relying on more stuff that is built into the framework (which I didn't find quick analogs for on a good amount of the content).